src/components/Login.js

```diff
@@ -128,7 +128,7 @@ export const Login = () => {
             >
               {isSubmitting ? (
                 <>
-                  <CircularProgress size={14} />
+                  <Loader size={14} />
```

Comment:
you can actually keep `CircularProgress` here instead of using `Loader` since the previous implementation was not using a `div` with a `className` of `loading`

Reply:
You can revert this one as well